packages/discord.js/.eslintrc.json

```diff
@@ -19,17 +19,6 @@
         }
       }
     ],
-    "prettier/prettier": [
-      2,
-      {
-        "printWidth": 120,
-        "singleQuote": true,
-        "quoteProps": "as-needed",
-        "trailingComma": "all",
-        "endOfLine": "lf",
-        "arrowParens": "avoid"
-      }
-    ],
```

Comment:
The discord.js package uses an old version of our prettier setup, can't just remove that yet sadly.

Reply:
![image](https://user-images.githubusercontent.com/74945038/172355154-cb0da7fa-88fd-4b18-a299-e2a99f94a79b.png)

This from `WebSocket.js` file

Reply:
Alright. Guess we keep it then.